spark/src/main/scala/io/delta/sharing/spark/RemoteDeltaFileIndex.scala

```diff
@@ -151,10 +152,13 @@ private[sharing] case class RemoteDeltaCDFAddFileIndex(
   override def listFiles(
     partitionFilters: Seq[Expression],
     dataFilters: Seq[Expression]): Seq[PartitionDirectory] = {
+    val updatedFiles = addFiles.map { a =>
+      AddFileForCDF(a.url, a.id, a.getPartitionValuesInDF, a.size, a.version, a.timestamp, a.stats)
```

Comment:
Question: makePartitionDirectories will call getPartitionValuesInDF again? Would that lead to double inclusion of the cdf columns?
Consider adding some comments.

Reply:
It's not double inclusion, and actually to make them consistent. Here, it's applied to each file, makePartitionDirectories is applying the partitions to the directory path.